carball/json_parser/actor/game_event.py

```diff
@@ -0,0 +1,21 @@
+from .base import *
+
+
+class GameEventHandler(BaseActorHandler):
+    type_name = 'Archetypes.GameEvent.GameEvent_Soccar'
```

Comment:
Is this used for anything? Or just here for future reference.

Reply:
I probably left it there by accident